src/PyListProxyHandler.cc

```diff
@@ -1316,7 +1315,7 @@ static uint32_t FlattenIntoArrayWithCallBack(JSContext *cx,
       return false;
     }
 
-    PyObject *element = pyTypeFactory(cx, retVal)->getPyObject();
+    PyObject *element = pyTypeFactory(cx, retVal);
```

Comment:
`element` needs to be `Py_DECREF`'d before the end of this loop iteration

Reply:
done